Statsman/ChartsService.cs

```diff
@@ -24,7 +24,7 @@ public async Task<Stream> GetImageStatisticsPerPeriod(IEnumerable<TimeStatisticI
             var labels = statistics.Count() switch
             {
                 int x when x <= 1 => dates.Select(x => x.ToString("yyyy-MM-dd HH")),
-                int x when x <= 30 => dates.Select(x => x.ToString("yyyy-MM-dd")),
+                int x when x <= 31 => dates.Select(x => x.ToString("yyyy-MM-dd")),
                 int x when x > 30 => dates.Select(x => x.ToString("yyyy-MM")),
```

Comment:
\>30 ? chyba powinno być \>31
nie ma pojęcia co tutaj jest robione, więc tylko zapytam, czy teraz to będzie działać też dla miesięcy które mają 30 albo 28 dni?
wcześniej to działało bo pewnie pisałeś w miesiącu który miał 30 dni, a głupio by było co miesiąc to poprawiać xD

Reply:
druga rzecz to sprawdź czy da się użyć tego uproszczonego switcha z c# 9 (bez int x when x)

Reply:
`nie ma pojęcia co tutaj jest robione` => format wyświetlania dat na wykresie
jak wyświetlasz dane dla np połowy roku
to nie chcesz żeby pojawiały sie informacje o dniach
chcesz tylko informacje o miesiącach

Reply:
poprawione